cli/io/testconfig.json

```diff
@@ -3,6 +3,9 @@
     "http://some.site:8081": "dXNlcjE6cGFzc3dvcmQx",
     "http://another.one:8081": "dXNlcjI6cGFzc3dvcmQy"
```

Comment:
maybe `dXNlcjI6cGFzc3dvcmQy` should look more like a JWT for this site (authType Bearer)